index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Virtual World Demo</title>
+    <title>Comms Web Demo</title>
```

Comment:
Do you want to add "Dolby.io" in the title?

Reply:
Something like that: Comms Web Demo - Dolby.io ?

Reply:
I would do the other way `Dolby.io - Comms Web Demo` so it always shows on the tab